gadgets/trace_dns/test/integration/trace_dns_test.go

```diff
@@ -56,6 +57,16 @@ type traceDNSEvent struct {
 	RecursionAvailable bool   `json:"ra"`
 }
 
+const (
+	DefaultServerImage = "ghcr.io/inspektor-gadget/dnstester:latest"
+	DefaultClientImage = "docker.io/library/busybox:latest"
+)
+
+var (
+	ServerImage = DefaultServerImage
+	ClientImage = DefaultClientImage
```

Comment:
Are these global variables needed? IMO they can be moved as variables inside TestTraceDNS, and you can use something like this to initialize them:

```golang
	serverImage := os.Getenv("SERVER_IMAGE")	
	if serverImage == "" {
		serverImage = DefaultServerImage
	}
```